PRACTICE.md

```diff
@@ -157,5 +157,52 @@ Keep close track of both of these processes so that you can answer any questions
+
+In the MASQ Network project, the process of QA is triggered when a card is moved into the 'Quality Assurance In Progress' column on our [Card Wall](https://github.com/MASQ-Project/Node/blob/master/COLUMNS.md)
+
+The Testing Supervisor or QA Manager is responsible for passing or failing a card based on the feedback from the test team across the different operating systems being supported. Any feedback, bugs or suggestions are communicated to the developer of the card, and another iteration of development, testing and review is completed.
+
+CLI-based QA requires some basic command-line knowledge and understanding of the Node software itself.
+
+If you would like to contribute to QA testing, and have a good foundational knowledge of CLI, please reach out to our team by [email](mailto:info@masq.ai) or join our [Discord](https://discord.gg/masq) and tag one of our admins.
+
+## Further Information on Versioned Releases
+The determination of versioning will start with a discussion of the core developers and the Product Owner. This will begin towards the end of a cards engineering practice above, most likely during final review and QA steps.
+
+Versioning follows basic [semver format](https://semver.org) - view the formal documentation and conventions for reference.
+
+The core specifications to be followed are:
+
+- Once a versioned package has been released, the contents of that version MUST NOT be modified. Any modifications MUST be released as a new version.
+- A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphens [0-9A-Za-z-]. Identifiers MUST NOT be empty. Numeric identifiers MUST NOT include leading zeroes. Pre-release versions have a lower precedence than the associated normal version. A pre-release version indicates that the version is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92, 1.0.0-x-y-z.–.
+
+### Steps Followed for Version Increment
+Once the discussion around the development/feature card has completed, there will be a git tag added to the merge commit of the branch when it is merged to `master` by an admin user of the public repo.
+If the merged card is not determined to be included in an immediate release with version increment, then the git tag will be the current version number, denoted with `pre-release.1` or another suitable qualifier following the patch number, eg `v0.6.0-pre-release.1`. It may become common, that there are several cards being merged into `master` under pre-releases before the next version increment is formally tagged and released. In this case the git tag will be incremented with a dot separated number that increases each time, so there is never a duplicate git tag.
```

Comment:
What I hoped for that I would read in this documentation is some sort of show of particular steps that you need to take in order to create the tag.

I thought a bit about it now and although I don't have very good knowledge of that I'd add something like this:

"About technical aspects of the tag. We think we need only the so-called lightweight tag, that is without an extend in the form of annotation. A possibly right way of taking care of the release tag is first to go to acquire an agreement from the product owner that your branch (in your open PR) is going to enclose the current, pending release. If this is the case you want to make sure your code in your branch appears errorless in Actions (it's passed all checks) and also it's satisfied all reviews and so is approved by the reviewer. 

At that point you create a tag to the HEAD commit of your branch with the previously agreed version number. 
If you do that locally on your machine you may want to use e.g.:

`git tag v0.6.1
git push origin v0.6.1`

The last step should cause a new trigger of a run in Actions, this time with a different workflow than usual, producing official release binaries at the end. 
 
Beside that, while you know the tag is in its place you're finally ready to let the merge button for your PR to master be pressed."

Maybe somewhere here I'm not accurate or even wrong in the steps so please think it over together with Finsaas. ;)

Reply:
great I have added this with a few edits to your outline

Thanks @bertllll !